test/generator/tiled_blur_blur_generator.cpp

```diff
@@ -2,9 +2,18 @@
 
 namespace {
 
+template<int channels = 3>
```

Comment:
This seems like it could just be an argument with a default value, not a template parameter.